addons/mail/static/src/models/thread/thread.js

```diff
@@ -1462,6 +1462,14 @@ function factory(dependencies) {
             return replace(this.activities.filter(activity => activity.state === 'planned'));
         }
 
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeHasCallFeature() {
+            return this.rtcSessions.length === 0 && ['channel', 'chat', 'group'].includes(this.channel_type);
```

Comment:
The `this.rtcSessions.length === 0` part should stay on the template for minimal diff, and also the field "call feature" is generic so it shouldn't depend on whether there is already a call in progress.